core/pom.xml

```diff
@@ -472,9 +472,9 @@ THE SOFTWARE.
       <scope>test</scope>
     </dependency>
     <dependency><!-- needed by Jelly -->
-      <groupId>javax.servlet</groupId>
-      <artifactId>jstl</artifactId>
-      <version>1.1.0</version>
+      <groupId>javax.servlet.jsp.jstl</groupId>
```

Comment:
Any risk of compatibility issues in plugins?

Reply:
there is always a risk - but 1.2 has been part of the Java EE spec since Java EE 5 (may 2006) that contained Servlet 2.5 so you likely got this at run time anyway.  Whilst 1.2.1 is not called out in any spec it is an old version (also of the 2006 era in the JCP)
Maybe we should ask the [spec lead](https://jcp.org/en/jsr/detail?id=52) who is now involved in the Jenkins community :-)
/cc @pelegri